mrtt-ui/src/App.js

```diff
@@ -114,6 +116,8 @@ function App() {
             <Route path='/auth/signup' element={<SignupForm />} />
             <Route path='/auth/login' element={<LoginForm />} />
             <Route path='/auth/login/newUser' element={<LoginForm isUserNew={true} />} />
+            <Route path='auth/forgot-password' element={<ForgotPasswordForm />} />
+            <Route path='auth/password/reset/' element={<ResetPasswordForm />} />
```

Comment:
I got this reset path from the path that the email used. Its inconsistent, but tolerable given our constraints I think. Trying to save a back and forth.

Reply:
Angelo made an assumption. I dont want to engage in one of those back and forths.